kubetools/dev/backends/docker_compose/docker_util.py

```diff
@@ -137,7 +138,12 @@ def get_containers_status(
             env = compose_project.replace(docker_name, '')
 
         # Where the name is compose-name_container_N, get container
-        name = container.name.split('_')[1]
+        print(container.name)
+        print(container.labels.get('kubetools.project.env'))
+        if int(__version__.split('.')[0]) < 20:
+            name = container.name.split('_')[1]
+        else:
+            name = container.name.split('-')[1]
```

Comment:
Seems like a useless check, this code will only ever be used with this version of the tool, and `__version__` is AFAIK the version of the tool, nothing else

Reply:
> I doubt this is going to work as-is. It would be very lucky that all of the features of docker-compose that kubetools uses are fully backwards-compatible in docker compose.

this is why I have stuck this in draft, it still needs a full test across a range of OSes, archs and with as many of our services as possible.

Reply:
Also still has print statements! I will remove those alongside the check

Reply:
[a439c4c](https://github.com/EDITD/kubetools/pull/163/commits/a439c4cf81449d620986cec9e67d3119f08ba093)